scholarly/_scholarly.py

```diff
@@ -255,13 +258,29 @@ def citedby(self, object: Publication)->_SearchScholarIterator:
         :param object: The Publication object for the bibtex exportation
         :type object: Publication
         """
-        if object['container_type'] == "Publication":
-            publication_parser = PublicationParser(self.__nav)
-            return publication_parser.citedby(object)
-        else:
+
+        if object['container_type'] != "Publication":
             self.logger.warning("Object not supported for bibtex exportation")
             return
 
+        if object["bib"]["citedby"] < 999:
+            return PublicationParser(self.__nav).citedby(object)
+        else:
+            try:
+                year_low = int(object["bib"]["pub_year"])
+                year_end = int(datetime.date.today().year)
+            except KeyError:
+                self.logger.warning("Unknown publication year for paper %s, may result in incorrect number of citedby papers.", object["bib"]["title"])
+                return PublicationParser(self.__nav).citedby(object)
+
+            pub_id = int(object["citedby_url"].split("=")[1].split("&")[0])
+            iter_list = []
+            while year_low < year_end:
+                iter_list.append(self.search_citedby(publication_id=pub_id, year_low=year_low, year_high=year_low+1))
+                year_low += 1
+
+            return itertools.chain(*iter_list)
```

Comment:
Using `yield from` syntax would be much cleaner and would avoid importing `itertools`.